sdcm/results_analyze/__init__.py

```diff
@@ -443,6 +443,9 @@ def _calculate_relative_change_magnitude(current_value, best_value):
 
     def check_regression(self, test_id, data, is_gce=False, node_benchmarks=None, email_subject_postfix=None):  # pylint: disable=too-many-locals, too-many-branches, too-many-statements, too-many-arguments  # noqa: PLR0914
```

Comment:
```suggestion
    def check_regression(self, test_id, data, is_gce=False, node_benchmarks=None, email_subject_postfix=None) -> None:  # pylint: disable=too-many-locals, too-many-branches, too-many-statements, too-many-arguments  # noqa: PLR0914
```

also remove the `return True` from the end of the function

Reply:
think at it again, it's just nitpick...